rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -1618,6 +1621,28 @@ void settings_dialog::EnhanceSlider(emu_settings::SettingsType settings_type, QS
 	}
 }
 
+void settings_dialog::SnapSlider(QSlider *slider, int interval)
+{
+	connect(slider, &QSlider::sliderPressed, [this, slider]()
+	{
+		m_currentSlider = slider;
```

Comment:
I think you can use sender() instead of the capture

Reply:
I can't get sender() to work from within the lambda, it just crashes the whole application.

Reply:
Quick google search seems to suggest that sender isn't compatible with lamdas and should only used with slot methods.